ui/package.json

```diff
@@ -49,12 +49,14 @@
     "@fontsource/quicksand": "^4.4.5",
     "@fortawesome/fontawesome-free": "^6.0.0",
     "@stomp/stompjs": "^6.1.2",
+    "@types/react-beautiful-dnd": "^13.1.2",
```

Comment:
do we need types in dependencies or should that be in devDependencies?

Reply:
Nope. Just moved.